tasks/drupal.yml

```
@@ -65,12 +65,7 @@ tasks:
       - ./vendor/bin/drush {{.site}} --yes config:import
       - ./vendor/bin/drush {{.site}} --yes updatedb --no-cache-clear
       - ./vendor/bin/drush {{.site}} --yes cache:rebuild
-      - |
-        if [[ $(./vendor/bin/drush config:status --format=json --state=Different) != '[]' ]]; then
-          echo "Config export does not match database."
-          ./vendor/bin/drush config:status
-          exit 1;
-        fi
+      - task: :test:config
```

Fixing this typo and then hopefully things pass!

```suggestion
      - task: test:config
```